ImageData/HDF5Attributes.h

```diff
@@ -2,22 +2,25 @@
 #ifndef CARTA_BACKEND_IMAGEDATA_HDF5ATTRIBUTES_H_
 #define CARTA_BACKEND_IMAGEDATA_HDF5ATTRIBUTES_H_
 
+#pragma once
+
 #include <casacore/casa/Containers/Record.h>
 #include <casacore/casa/HDF5/HDF5HidMeta.h>
 
 class HDF5Attributes {
 public:
     // HDF5Record::doReadRecord modified to not iterate through links
     // (links get HDF5Error "Could not open group XXX in parent")
-    static casacore::Record doReadAttributes(hid_t groupHid);
```

Comment:
Is the "do" in this function name necessary? It's already a verb. :)

Reply:
Not necessary, started with casacore "doReadRecord" function.  Feel free to rename during code styling name changes.